annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -845,6 +896,7 @@ public void validateWebAnnotation(Annotation webAnnotation)
 	    validateTag(webAnnotation);
 	    break;
 	case TRANSCRIBING:
+	    validateTranscribing(webAnnotation);
 	    validateEdmRights(webAnnotation);
```

Comment:
validation of edmRights must be included in the vaidateTranscription method.

Reply:
Validation method for edmRights moved to validateTranscription